docs/src/main/sphinx/release/release-434.md

```diff
@@ -0,0 +1,36 @@
+# Release 434 (22 Nov 2023)
+
+## General
+
+* {{breaking}} Rename the `query.max-writer-tasks-count` configuration property
+  and the related `max_writer_tasks_count` session property to
+  `query.max-writer-task-count` and `max_writer_task_count`. ({issue}`19793`)
+* Prevent table function failure when its invocation is a first query run on a
+  worker. ({issue}`19849`)
+* Prevent sudden increases in PagePartitioner memory consumption in some
```

Comment:
PagePartitioner is an implementation detail. What's the user-visible condition that causes this? (e.g., specific query shapes, features, etc)

Reply:
PR is large .. but I reworded to mention affected queries.. we might have to hedge with "and similar scenarios" or so